vspace/src/main/java/edu/asu/diging/vspace/core/services/impl/ModuleLinkManager.java

```diff
@@ -104,4 +110,17 @@ protected void deleteLinkRepo(IModuleLink link) {
         moduleLinkRepo.delete((ModuleLink) link);
     }
 
+    @Override
+    public HashSet < ISpace > findModuleLinksFromModuleId(String moduleId) {
+        List < ModuleLink > moduleLinks = moduleLinkRepo.getModuleLinks(moduleId);
+        List < ISpace > spaces = new ArrayList < > ();
+        for (ModuleLink ml: moduleLinks) {
+            String spaceId = moduleLinkRepo.getSpaceIdFromModuleLink(ml.getId());
+            ISpace space = spaceManager.getSpace(spaceId);
+            spaces.add(space);
+        }
+        HashSet < ISpace > uniqueSpaces = new HashSet < ISpace > (spaces);
+        return uniqueSpaces;
+    }
+
```

Comment:
please fix your IDE settings  so there are no spaces around the `<`, `>`

Reply:
fixed